tests/baselines/reference/api/typescript.d.ts

```diff
@@ -14,7 +14,7 @@ and limitations under the License.
 ***************************************************************************** */
 
 declare namespace ts {
-    namespace server {
+    export namespace server {
```

Comment:
I don’t know what happened here. It occurred only after merging upstream into the branch.

Reply:
It usually means something isn't exported in the API so the bundler can't just rely on explicit exporting; you can diff the internal dts files before and after to see

Reply:
I know the cause then 😄